    protected Throwable getError() {
        return m_exception;
    }

